src/FormsSettings.vue

```diff
@@ -0,0 +1,167 @@
+<!--
+  - @copyright Copyright (c) 2021 Jonas Rittershofer <jotoeri@users.noreply.github.com>
+  -
+  - @author Jonas Rittershofer <jotoeri@users.noreply.github.com>
+  -
+  - @license AGPL-3.0-or-later
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<div>
+		<SettingsSection :title="t('forms', 'Form Creation')"
+			:description="t('forms', 'Decide who can create forms.')">
+			<CheckboxRadioSwitch ref="switchRestrictCreation"
+				:checked.sync="appConfig.restrictCreation"
+				class="forms-settings__creation__switch"
+				type="switch"
+				@update:checked="onRestrictCreationChange">
+				{{ t('forms', 'Restrict form creation to selected groups') }}
+			</CheckboxRadioSwitch>
+			<Multiselect v-model="appConfig.creationAllowedGroups"
+				:disabled="!appConfig.restrictCreation"
+				:multiple="true"
+				:options="availableGroups"
+				:placeholder="t('forms', 'Select groups')"
+				class="forms-settings__creation__multiselect"
+				label="displayName"
+				track-by="groupId"
+				@update:value="onCreationAllowedGroupsChange" />
+		</SettingsSection>
+		<SettingsSection :title="t('forms', 'Form Sharing')"
+			:description="t('forms', 'Configure which options are available for sharing a form.')">
+			<CheckboxRadioSwitch ref="switchAllowPublicLink"
+				:checked.sync="appConfig.allowPublicLink"
+				type="switch"
+				@update:checked="onAllowPublicLinkChange">
+				{{ t('forms', 'Allow sharing by Public link') }}
+			</CheckboxRadioSwitch>
+			<CheckboxRadioSwitch ref="switchAllowPermitAll"
+				:checked.sync="appConfig.allowPermitAll"
+				type="switch"
+				@update:checked="onAllowPermitAllChange">
+				{{ t('forms', 'Allow sharing to all logged in users') }}
```

Comment:
```suggestion
				{{ t('forms', 'Allow sharing to all logged in accounts') }}
```

This is to remove the term "user" from the interface language and to also ensure gender neutrality in other languages. (See https://help.nextcloud.com/t/geschlechtsneutrale-sprache-genderneutral/82200/107 (German))